<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Toast{
	background: transparent;rgb(72, 111, 141);
	border-width: 1px;
	border-color:black;
}*/</string>
  </property>
  <widget class="QWidget" name="toastBox" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*#toastBox {
	background-color: rgb(180, 50, 90,100);
}*/</string>
   </property>
   <widget class="QToolButton" name="btnToastExit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>8</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Toast &gt; QToolButton{
	background: transparent;
	opacity: .5;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Icon/Cross.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnToastUpIcon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Icon/up.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>186</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="qLabelMode">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: white;
	opacity: 1;
}
</string>
       </property>
       <property name="text">
        <string>Mode: WAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qLabelStatus">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Status: UP!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qLabelStatus_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Press this button to stop the alarm!</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
